src/runtime/pkg/containerd-shim-v2/create.go

```diff
@@ -19,6 +19,7 @@ import (
 	"strings"
 	"syscall"
 
+	"github.com/container-orchestrated-devices/container-device-interface/pkg/cdi"
```

Comment:
Change back to tags-cncf. This was the right one; this ^^ is the old implementation and location of CDI

Reply:
Both CDI-go and CDI-rs are hosted on cncf-tags github repo